x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/StartTrainedModelDeploymentAction.java

```diff
@@ -187,12 +187,6 @@ public long estimateMemoryUsageBytes() {
             return MEMORY_OVERHEAD.getBytes() + 2 * modelBytes;
         }
 
-        @Override
-        public String getWriteableName() {
```

Comment:
This is no longer a named writeable as the task params are (de)serialized directly in the allocation metadata.